forum/qa-plugin/report-reason/q2apro-flag-reasons-lang-default.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+return array(
+    'reason' => 'Pow&#243;d:',
+    'reason_spam' => 'Kod bez bloczka',
+    'reason_quality' => 'SPAM',
+    'reason_rude' => 'Duplikat',
+    'reason_edit' => 'Z&#322;e otagowanie',
+    'reason_duplicate' => 'Niepoprawna kategoria',
+    'reason_offtopic' => 'Obraza',
```

Comment:
Jak reason offtopic ma się do "obraza"? W ogóle jak to się stało, że te reasony nie mają nic wspólnego z tym co jest tłumaczeniem w języku polskim?

Reply:
Szczerze, nie chciało mi się wertować  całego kodu tylko po to, żeby zmienić nazwy w array których nikt nie widzi. Myślę, że to nie zaszkodzi. Przy całym Q2A to jest nic.

Reply:
"Przy całym Q2A to jest nic." nie jest żadnym argumentem.

Reply:
Szczerze mówiąc, te tablice są rozsiane po całym kodzie, w większości funkcji coś z nich jest, baza też z nich korzysta, i naprawdę wg mnie nie ma sensu tego zmieniać.